tabulator/__init__.py

```diff
@@ -3,10 +3,22 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+import io
+import os
+
+
+# General
 
 from .stream import Stream
 from . import exceptions
 
 # Deprecated
+
 from .topen import topen
 from .stream import Stream as Table
+
+# Version
+
+__version__ = io.open(
```

Comment:
What is the background for storing the version on a separate file rather than hardcoded here or on setup.py? Seems like a very convoluted way of defining it